### ISSUE TITLE
Import keys from geth

### ISSUE THREAD

**Issue by mongrim** [ethcore/parity#2268]
It fails on importing some keys from geth client:

```
2016-09-23 15:59:57 Import geth accounts failed. Invalid key file: "/home/user/.parity/keys/UTC--2016-03-15T14-50-06.361821586Z--78ba4a41b82a56991922b32ea90e0d2425f952c1": Invalid hash at line 1 column 164
Could not open keys directory: Invalid key file: "/home/user/.parity/keys/UTC--2016-03-15T14-50-06.361821586Z--78ba4a41b82a56991922b32ea90e0d2425f952c1": Invalid hash at line 1 column 164
```

Keys file is in attachment (here tried to import testnet (morden) keys, but it fails on mainnet too)

[keys.zip](https://github.com/ethcore/parity/files/489717/keys.zip)

**Comment by NikVolf** [comment]
duplicate of #2263 
ciphertext is 62 hex digits also

**Comment by yikwei91** [comment]
Hi, I'm not tech savy at all. Was using ethereum wallet, then trying to switch to parity. I didn't get how can i resolve this via #2263 ? What i did was install parity, fire it up with --geth, install wallet, and put in the keyfile into wallet directory. Then i get the error as above? Can anybody please explain to me?

**Comment by yikwei91** [comment]
2016-10-06 14:40:45 Import geth accounts failed. Invalid key file: "C:\\Users\\chee\\AppData\\Roaming\\Ethereum\\keystore\\UTC--2016-07-03T06-48-35.524417700Z--a538ee230098b92b1c1e2d81dcb3b4288c940b98": Invalid hash at line 1 column 164 
Error to be exact-**

**Comment by tomusdrw** [comment]
@yikwei91 The fix will be shortly released with 1.3.4 version of Parity.

**Comment by yikwei91** [comment]
thanks a lot @tomusdrw
